packages/browser/src/core/events/__tests__/index.test.ts

```diff
@@ -322,6 +322,48 @@ describe('Event Factory', () => {
         innerProp: '👻',
       })
     })
+
+    describe.skip('anonymousId', () => {
```

Comment:
These tests currently don't work? Are they regressions or existing issues?

Reply:
Not regressions, they are just existing edge cases / undefined behavior that I uncovered when working on this bug.